### ISSUE TITLE
Hexen 2 bsp support

### ISSUE THREAD

**Issue by andyp123** [andyp123/blender_io_mesh_bsp#12]
At least investigate this, since Hexen 2 has the same or similar BSP version.

**Comment by motorsep** [comment]
Yes please, I'd love to be able to import Hexen 2 BSPs into Blender (with textures) !!!

dmodel_t lump is different from Quake 1 if it helps any. 

Some info I gathered from FTEQW author:
"Its textures have an implicit palette exactly like q1 does. H2bsp's lava is opaque to vis while its water isn't, so I guess that's another difference. The only important one is that Hexen 2 bsp have 8 slots for hulls (6 used, iirc), while q1 has 4 (3 used)."

**Comment by andyp123** [comment]
Sorry for not replying sooner. Thanks for the info. As you can guess I've not gotten around to it, and not sure I ever will, but it's useful information if I or someone else ever does.

**Comment by motorsep** [comment]
I am still keeping fingers crossed for this one 😊
